src/presentational-components/shared/toolbar.js

```diff
@@ -69,13 +69,13 @@ export const filterConfigBuilder = (
   textFilters,
   sortBy
 ) => ({
-  items: [ ...textFilters && textFilters.length > 0 ? textFilters.map(({ key, value }) => ({
+  items: [ ...textFilters && textFilters.length > 0 ? textFilters.map(({ key, value, isExact }) => ({
     label: firstUpperCase(key),
     type: 'text',
     filterValues: {
       id: `filter-by-${key}`,
       key: `filter-by-${key}`,
-      placeholder: `Filter by ${key}`,
+      placeholder: `Filter by ${isExact ? 'exact ' : '' }${key}`,
```

Comment:
Change the prop not to be `isExact` but `placeholkder` and if set use it instead of generated placeholder.

Reply:
No problem, fixed :)